pyk/src/pyk/tests/test_kastManip.py

```diff
@@ -103,33 +104,39 @@ def test_minimize_term(self):
                 self.assertEqual(actual, expected)
 
 
-class MlPredToBoolTest(TestCase):
+class BoolMlPredConversionsTest(TestCase):
+
+    # TODO: We'd like for bool_to_ml_pred and ml_pred_to_bool to be somewhat invertible.
+    test_data_1 = (
```

Comment:
```suggestion
    test_data_ml_pred_to_bool : Iterator[Tuple[str, bool, ...]] = (
```

Reply:
Seems unnecssary, but I'll add.

Reply:
I actually can't get this typing to work.

Reply:
Recorded: https://github.com/runtimeverification/k/issues/2801